etc/bash/docker.sh

```diff
@@ -1,6 +1,26 @@
 #!/usr/bin/env bash
 
 cd $(dirname $0)/../docker
-docker-compose up -d
+cp .env.dist .env || echo "Previosuly created env"
+docker-compose up -d || docker-compose up -d
 docker-compose exec php bash -c "cd /var/www/taskmanager && composer install"
 docker-compose exec php bash -c "cd /var/www/identityaccess && composer install"
+docker-compose exec php bash -c "cd /var/www/identityaccess/var && chown www-data:www-data * && rm -rf cache/*"
+docker-compose exec php bash -c "cd /var/www/taskmanager/var && chown www-data:www-data * && rm -rf cache/*"
+docker-compose exec php bash -c "cd /var/www/identityaccess && sh etc/bash/generate_ssh_keys.sh"
+docker-compose exec php bash -c "echo '172.18.0.6 identityaccess.localhost' >> /etc/hosts"
+docker-compose exec php bash -c "echo '172.18.0.6 taskmanager.localhost' >> /etc/hosts"
+
```

Comment:
Not sure if it is always the same IP, maybe we can get server's current IP and set it there